### ISSUE TITLE
May I ask for the the knowledge of hyperparameters (eg. lr, scheduler, optimizer) during training.

### ISSUE THREAD

**Issue by ZihanLiao** [FireRedTeam/FireRedASR#63]
may ask if a sketch of hyperparameters can be shown.
